blog/2023-09-26-svelte-flow-alpha-xyflow.mdx

```diff
@@ -0,0 +1,111 @@
+
+# Soon: Fresh paint and changing our name 🎨
+
+<figure>
+  <img
+    src="/img/blog/2023-svelte-alpha-xyflow/xyflow-banner.png"
+    alt="A screenshot of a header of a page. The title says Wire your ideas with xyflow."
+  />
+  <figcaption>A sneak peek of the new website</figcaption>
+</figure>
+
+Once Svelte Flow started to materialize, we quickly realized “React Flow” is an inflexible name. Just imagine “npm install reactflow/svelte” 😬😬
+
+“xyflow” will soon be the roof that React Flow and Svelte Flow live under. We like that “xy” can represent a variable (”xy” can be replaced with “react” or “svelte” to create the names of the individual libraries), as well as the xy coordinates of a graph. And you pronounce it like… erm… we’ll have to get back to you on that one.
+
+There will be package name changes as a result, and we’ll give clear instructions on how to make this change as soon as we “officially” launch all of this.
+
+And all of this is going to come with a fresh website, thanks to the brilliant [Facu Montanaro](https://facumontanaro.com/) 👏🏻
+
+# We went into the real world! Ahhh
+
+Hayleigh went to DevRelCon in London, where she brought back a bunch of findings on how we can better help folks using React Flow and the ecosystems around it. The videos from the conference should be dropping soon [here](https://developerrelations.com/event/devrelcon-london-2023).
+
+John gave another talk about money in open source in Portland, Oregon at FOSSY. This one’s 20 minutes with Q&A, so it’s our most comprehensive share on the topic yet. You can [watch it here](https://archive.org/details/fossy2023_Dear_Open_Source_lets_do_a_bet).
+
+Moritz and John held the first React Flow “Town Hall,” where we opened up our Discord for live questions from anyone who wanted to ask. We were looking for a way to be transparent about what’s coming up next at React Flow, without sticking to the structure of a roadmap. We’ll definitely be doing it again, it was a lot of fun!
+
+# Open Sourcing our Pro Platform 🎁
+
+Our Pro platform allows subscribers to access the code and tutorials of our "Pro Examples", but the repository for this platform has always been private. On our quest to allow more open source projects to be fairly compensated for their work, Christopher has been refactoring our Pro Platform from the ground up.
+
+<figure>
+  <img
+    src="/img/blog/2023-svelte-alpha-xyflow/dashboard.jpeg"
+    alt="A screenshot with a code example called Force Layout, with a tutorial below it."
+  />
+  <figcaption>Our Pro Platform, soon to be open source</figcaption>
+</figure>
+
+We hope other companies like us might use it in the future to build businesses around their open source software. If you're interested in this, please reach out to us, we'd love to know how this could work for you.
+
+# Whew.
+
+Ok that was a lot. Thanks for letting us get all of that off our chest. We’ll be posting again soon once we do an official launch of _gestures broadly at everything_
```

Comment:
whats _gestures broadly at everything_

Reply:
determined that this is tough to understand for a bunch of readers (non-native english folks), changed to something more universal